.github/workflows/daily-cleanup.yml

```diff
@@ -76,28 +74,11 @@ jobs:
           tf-bucket: "${{ env.TF_S3_BUCKET }}"
           max-age-hours-cluster: "${{ env.MAX_AGE_HOURS_CLUSTER }}"
 
-      - name: Close issue in case of success
-        shell: bash
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          WORKFLOW_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-        run: |
-          issue_url=$(gh --repo "${{ github.repository }}" search issues  "Failed to cleanup clusters" --state=open --label="schedule" --json 'url' | jq '.[0].url' -r)
-
-          if [ "$issue_url" != "null" ]; then
-            gh issue close --comment "Auto-closing issue as the cleanup has been successfuly ran in ${{ env.WORKFLOW_URL }}." "$issue_url"
-          else
-            echo "No open issues found with the specified criteria."
-          fi
-
-      - name: Create issue on failure
-        if: failure()
-        uses: JasonEtco/create-an-issue@1b14a70e4d8dc185e5cc76d3bec9eab20257b2c5 # v2
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          WORKFLOW_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+      - name: Notify in Slack in case of failure
+        id: slack-notification
+        if: failure() && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
```

Comment:
do we care about the `workflow_dispatch` even being reported?
May be someone trying something out and is usually triggered with intention.